### PR TITLE
[15.0][MIG] hr_commission: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa TT36956

### CODE REVIEW

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza @victoralmau please review

File: hr_commission/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Copyright 2021 Tecnativa - João Marques
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "HR commissions",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/commission",
+    "category": "Commissions",
+    "depends": ["sale_commission", "hr"],
```
Comment (pedrobaeza) [inline hr_commission/__manifest__.py]: No need of depending on the sales one now.

```suggestion
    "depends": ["account_commission", "hr"],
```

File: hr_commission/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2015-2020 Tecnativa - Pedro M. Baeza
+# Copyright 2021 Tecnativa - João Marques
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "HR commissions",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline hr_commission/__manifest__.py]:
```suggestion
    "author": "Tecnativa,Odoo Community Association (OCA)",
```

File: hr_commission/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+* `Tecnativa <https://www.tecnativa.com>`_
+
+  * Pedro M. Baeza
+  * João Marques
```
Comment (pedrobaeza) [inline hr_commission/readme/CONTRIBUTORS.rst:4]: Add yourself as contributor:

```suggestion
  * João Marques
  * Ernesto García
```

File: hr_commission/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+This module links sale_commission with hr module. For now, it only adds another
```
Comment (pedrobaeza) [inline hr_commission/readme/DESCRIPTION.rst:1]:
```suggestion
This module links account_commission with hr module. For now, it only adds another
```

**Review by pedrobaeza — COMMENTED** [review]
Please switch the icons to these new modern ones:

icon
icon

**Comment by ernesto-garcia-tecnativa** [comment]
Done @pedrobaeza , please review

**Review by pedrobaeza — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]
Code review OK.

**Pull request merged**